api/jobs/jobs.py

```diff
@@ -83,10 +125,14 @@ def load(cls, e):
         # Don't modify the map
         d = copy.deepcopy(e)
 
-        if d.get('inputs', None):
-            inputs = d['inputs']
-            for x in inputs.keys():
-                inputs[x] = create_filereference_from_dictionary(inputs[x])
+        if d.get('inputs'):
```

Comment:
I take it the `, None` that you removed from this `get` call is unnecessary?
My memory of these map semantics is hazy.